timesketch/lib/graphs/win_services.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Graph plugin for Windows services."""
+
+from timesketch.lib.graphs.interface import BaseGraphPlugin
+from timesketch.lib.graphs import manager
+
+
+class WinServiceGraph(BaseGraphPlugin):
+    """Graph plugin for Windows services."""
+
+    NAME = 'WinService'
+    DISPLAY_NAME = 'Windows services'
+
+    def generate(self):
+        query = 'event_identifier:7045'
```

Comment:
better to have as `event_identifier:"7045"` ??

Reply:
hmm, good question. the type is INT in the mapping from Plaso, that's why I went with that.